detect-pr-changes.sh

```diff
@@ -13,11 +13,15 @@ mkdir tmpgit/new
 dotnet build -nologo -clp:NoSummary -v quiet tools/Google.Cloud.Tools.CompareVersions
 dotnet build -nologo -clp:NoSummary -v quiet tools/Google.Cloud.Tools.ReleaseManager
 
+log_header() {
+  echo "\e[1;35m$1\e[0m"
+}
+
 for api in $apis
 do  
   if [[ -d tmpgit/apis/$api/$api && -d apis/$api/$api ]]
   then
-    echo "Building $api"
+    log_header "Building $api"
```

Comment:
Maybe add a comment explaining what this does?

Reply:
Yup, will add it to the log_header definition.